python/ray/serve/_private/common.py

```diff
@@ -292,3 +292,21 @@ class RunningReplicaInfo:
     actor_handle: ActorHandle
     max_concurrent_queries: int
     is_cross_language: bool = False
+
+    def __post_init__(self):
+        # Set hash value when object is constructed
```

Comment:
I think we should add a comment emphasizing what's special about the hash function (the key feature is `actor_id` is used instead of `actor_handle`) and why it's necessary, so that a new developer doesn't have to trace the reasoning back to the PR description.  

Could be added here and/or in `compute_iterable_delta`, or wherever it makes sense based on your debugging

Reply:
done!